jobserver/forms.py

```diff
@@ -32,10 +32,15 @@ def __init__(self, actions, *args, backends=None, **kwargs):
             },
         )
 
-        if backends:
-            self.fields["backend"] = forms.ChoiceField(
-                choices=backends, widget=forms.RadioSelect
-            )
+        try:
+            initial = backends[0][0]
+        except IndexError:
+            initial = None
```

Comment:
When would this `IndexError` be hit?  If a user has no backends, should they ever see this form?  And if they shouldn't see this form but do somehow, should it blow up?

Reply:
> When would this IndexError be hit?

when there are no backends 😜🥁

This can probably go away once the current rush of changes is done!